tools/deployment/make_linux_package.sh

```diff
@@ -13,11 +13,11 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 SOURCE_DIR="$SCRIPT_DIR/../.."
 BUILD_DIR=${BUILD_DIR:="$SOURCE_DIR/build/linux"}
 
-export PATH="$PATH:/usr/local/bin"
+export PATH="/usr/local/osquery/bin:/usr/local/bin:$PATH"
 source "$SOURCE_DIR/tools/lib.sh"
 
 PACKAGE_VERSION=`git describe --tags HEAD || echo 'unknown-version'`
-PACKAGE_ARCH=`uname -m`
+PACKAGE_ARCH="x86_64"
```

Comment:
Good call, but there are a few other mechanics limiting us to `x86_64`, there's no way you can get this far without an Intel 64bit machine. I'd like to change that eventually, but for now this is only to record the "arch" that RPMs like to stick into their package metadata.